tests/test_Kratos_IO.py

```diff
@@ -1,41 +1,47 @@
-
 import json
 
 from stem.kratos_IO import KratosIO
-from stem.load import (PointLoad, Load)
-
+from stem.load import PointLoad, MovingLoad, Load
 from tests.utils import TestUtils
 
 
 class TestKratosIO:
-
     def test_create_load_process_dictionary(self):
         """
         Test the creation of the load process dictionary for the
         ProjectParameters.json file
         """
         # define load(s) parameters
-        point_load_parameters = PointLoad(active=[True, True, True],
-                                          value=[1000, 0, 0] )
+        # point load
+        point_load_parameters = PointLoad(active=[True, True, True], value=[1000, 0, 0])
```

Comment:
I think its better to set "active" to non-default values, same for the moving load parameters

Reply:
Agreed